src/screens/drafts/screen/draftsScreen.js

```diff
@@ -10,7 +10,7 @@ import { catchDraftImage } from '../../../utils/image';
 import { getFormatedCreatedDate } from '../../../utils/time';
 
 // Components
-import { BasicHeader, TabBar, PostListItem, PostCardPlaceHolder } from '../../../components';
```

Comment:
We are not using PostListItem anywhere else now that warrant name change to draftlistitem?